test/unit/index.spec.js

```diff
@@ -0,0 +1,9 @@
+var cfg = require('../../lib/config')
+
+describe('index', () => {
+  var index = require('../../lib/index')
+
+  it('should expose the `config` object', () => {
+    expect(index.config).toBe(cfg)
```

Comment:
Needs to be`.to.be.eql(` as we use chai not jasmine :) (that's why the tests are failing)